app/views/searches/dynamic.html.erb

```diff
@@ -0,0 +1,83 @@
+<% content_for (:head) { javascript_include_tag "dynamic_search" } %>
```

Comment:
The dynamic search page is complete and functional, but I have not added a link to the page to any of the menus.  At this point, the page can be used as a beta-testing ground, or @jywarren can decide to add a link and put it into play.